plugins/modules/cloud/packet/packet_device.py

```diff
@@ -46,6 +46,12 @@
     description:
       - List of device IDs on which to operate.
 
+  tags:
+    description:
+      - List of device tags,
+      type: list
+      elements: str
```

Comment:
```suggestion
    description:
      - List of device tags,
    type: list
    elements: str
```
i think sanity tests failed because of that